frontend/src/features/permits/components/form/ContactDetails.tsx

```diff
@@ -47,18 +53,33 @@ export const ContactDetails = ({ feature }: { feature: string }) => {
               name: "permitData.contactDetails.phone1",
               rules: {
                 required: { value: true, message: requiredMessage() },
+                validate: {
+                  validatePhone: (phone: string) =>
+                    (phone.length >= 10 && phone.length <= 20) ||
+                    invalidPhoneLength(10, 20),
+                },
               },
+
               label: "Phone Number",
               width: PHONE_WIDTH,
             }}
           />
 
           <CustomFormComponent
-            type="input"
+            type="number"
             feature={feature}
             options={{
               name: "permitData.contactDetails.phone1Extension",
-              rules: { required: false },
+              rules: {
+                required: false,
+                validate: {
+                  validateExt1: (ext?: string) =>
```

Comment:
This validation rule can be simplified to:
```
!ext || (ext.length === 0) || (ext.length <= 5) || invalidExtensionLength(5)
```
The other contact forms (in profile and elsewhere that uses optional extensions and phone numbers) can use this updated simplified rule as well